x-pack/elastic-agent/pkg/agent/cmd/enroll.go

```diff
@@ -384,3 +407,19 @@ func mapFromEnvList(envList []string) map[string]string {
 	}
 	return m
 }
+
+func getRandomPort() (uint16, error) {
```

Comment:
This works on all os's?  Seems risky to me.  Assuming that the OS will allow a rebind in a short period of time and/or won't decide to be aggressive and reuse port elsewhere.

Can we just default to 8221 or something?

Reply:
i was thinking also about reversal flow, and if we agree that having this configurable is ok. 
8221 is ok as well, seems it's not used by a lot of services. initially i wanted to avoid low ports, but i'm ok with this as well